gtests/test_nopython_callprofiler.cpp

```diff
@@ -5,7 +5,7 @@
 #error "Python.h should not be included."
 #endif
 
-#define CALLPROFILER 1
+#define MM_PROFILE 1
```

Comment:
This should be renamed to `MODMESH_PROFILE`.  I've made it very clear in [my previous comment](https://github.com/solvcon/modmesh/pull/559#pullrequestreview-3074094492):

> Rename the macro `CALLPROFILER` as `MODMESH_PROFILE` and the names added in cmake and make files.

Please let me know if I was not clear enough.